ui/app/routes/dynamic_evaluations/projects/$project_name/DynamicEvalRunSelector.tsx

```diff
@@ -147,7 +174,8 @@ export function DynamicEvalRunSelector({
                         const isSelected = selectedRunIds.includes(info.id);
                         const variantColor = getColor(info.id);
                         const runIdSegment = getLastUuidSegment(info.id);
-                        const isDisabled = !isSelected && !canAddMore;
+                        const isDisabled =
```

Comment:
Instead of showing a loading indicator every time we execute the fetcher, it's a little less janky if we just disable the items until the query has resolved. Kinda nice that the user can tell that the request is busy so they don't accidentally try to click something and hit the wrong option due to timing issues.

Reply:
This is neat!